tests/tensor/test_math.py

```diff
@@ -2187,8 +2189,9 @@ def test_broadcastable1(self):
         rng = np.random.default_rng(seed=utt.fetch_seed())
         x = TensorType(dtype=config.floatX, shape=(1, None, None))("x")
         y = tensor3("y")
-        z = tensordot(x, y)
+        z = tensordot(x, y, axes=2)
         assert z.broadcastable == (True, False)
+        assert z.type.shape == (1, None)
```

Comment:
This is redundant with the previous assert

Reply:
I actually wanted to see if the `type.shape` identified the second entry as `None` or not.

Reply:
I'll remove it anyway, since I've added a test that actually looks at the type shape

Reply:
I don't mind it though